app/assets/stylesheets/application.scss

```diff
@@ -11,11 +11,12 @@
  * It is generally better to create a new file per style scope.
  *
  *= require_self
+ *= require simple_calendar
```

Comment:
I think generally you want `require_self` underneath requiring any outside assets--you want whatever you write in `application.scss` to overwrite anything from an outside vendor.